Food_Tracker_App/app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I didn't end up using this file, but I'll probably update the textview items with this style once i finalize layout of the ingredient row